constructor/header.sh

```diff
@@ -494,8 +494,14 @@ fi
 
 # the second binary payload: the tarball of packages
 printf "Unpacking payload ...\n"
+{%- if bootstrap_with_tar %}
+{#- we only use tar for conda-standalone onedir builds, because otherwise conda cannot extract itself #}
+extract_range "${boundary1}" "${boundary2}" | \
```

Comment:
Since we know which internal files we have via `copy_conda_exe`, could we extract them the same way we extract `_conda`? For example, via a Jinja-templated for-loop? That would avoid bootstrapping with `tar`.

Reply:
> I dove in the git history but this code is there from the beginning (>12 years) so I wonder if those design constraints to not rely on tar still apply today.

That is a good question. It is plausible that this feature was reused because of Windows. In either case, there is no reason for the if condition. If the design decisions still hold, we shouldn't use `tar`. If they don't, we can just use `tar` in both cases.

I do see the advantage of using the `conda-standalone` process because it can apply data filters.